README.md

```diff
@@ -24,7 +24,7 @@ Feathr automatically computes your feature values and joins them to your trainin
 
 ## Documentation
 
-For more details, read our [documentation](https://linkedin.github.io/feathr/).
+For more details such as Feathr Concepts, read our [documentation](https://linkedin.github.io/feathr/).
```

Comment:
this doesn't make sense to. Why do we need to put Feathr concepts here?

If you want to highlights a few things for discovery, you should probably highliths  a few things with link?